README.md

```diff
@@ -49,6 +49,30 @@ new ShareButton({
 });
 ```
 
+## NPM installation
+
+1. ```npm i --save-dev share-button```
```

Comment:
just a quick, small, change: switch to one back tick instead of three for the single & inlined code